curriculum/js1/4.js

```diff
@@ -6,8 +6,10 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (num,input,result="",i=0) => {
```

Comment:
`i` is redundant. Just decrement num to 0

Reply:
I'm not clear on how to do that without affecting my code. Can you please help?